widget/slider.go

```diff
@@ -121,11 +127,21 @@ func (s *Slider) positionChanged(lastValue, currentValue float64) {
 
 	s.Refresh()
 
+	s.pendingChange = true
 	if s.OnChanged != nil {
 		s.OnChanged(s.Value)
 	}
 }
 
+func (s *Slider) checkInvokeOnChangeEnded() {
```

Comment:
Maybe something simpler like `fireChangeEnded`?

Reply:
What about `checkFireChangeEnded`? I like 'check' or similar being part of the function name to emphasize that if there is no pending change, it will not invoke the callback. Thoughts?

Reply:
I know what you mean but the word "check" is ambiguous to convey that.
It also checks that there is a OnChangeEnded - so which does it refer to.
Does it also require a version that will call the fire even if nothing has changed?

Reply:
Good point, I am fine with your first suggestion. Will update

Reply:
updated